tests/helpers/sql.ts

```diff
@@ -108,6 +108,7 @@ export async function createUserTeamAndOrganization(
         updated_at: new Date().toISOString(),
         personalization: '{}',
         setup_section_2_completed: true,
+        for_internal_metrics: false,
```

Comment:
Why is this needed if the module defaults to false? https://github.com/PostHog/posthog/blob/master/posthog/models/organization.py#L83

Reply:
.. because Django sets the defaults on the Model object, not the database, so anyone else using raw SQL will bork things :/

Reply:
wow, it's been on their minds (in some shape or form) for ~15 years: https://code.djangoproject.com/ticket/470